src/css/proposal.css

```diff
@@ -3,6 +3,7 @@
 }
 
 .proposal-wrapper {
+    overflow: hidden;
```

Comment:
Instead of adding an overflow: hidden scale down a little backcard-1 and backcard-2, e.g. scale(0.9). I checked it and on screen with width 320px it prevents showing the scrollbar

Reply:
Used and checked scale 0.94 for 320 px